python/sqlflow_submitter/explainer.py

```diff
@@ -14,13 +14,22 @@
 import sys
 
 import matplotlib
+# The default backend
 import matplotlib.pyplot as plt
+import sqlflow_submitter.pai.utils as utils
 
 # TODO(shendiaomo): extract common code from tensorflow/explain.py and xgboost/explain.py
 # TODO(shendiaomo): add a unit test for this file later
 
 
-def plot_and_save(plotfunc, filename='summary'):
+def plot_and_save(plotfunc,
+                  filename='summary',
```

Comment:
How about move `filename='summary'` to the last parameter, so we needn't specify it at every call site